src/components/ReviewProposals/ReviewProposals.tsx

```diff
@@ -0,0 +1,57 @@
+import { ColumnOptions, Link, Table } from "@sovryn/ui";
+import { ProposalType, useActiveBitocracyProposals } from "../../hooks/useActiveProposals";
+import { useMemo } from "react";
+import { truncate } from "../../lib/helpers";
+import { VetoAction } from "../VetoAction/VetoAction";
+import dayjs from "dayjs";
+import relativeTime from 'dayjs/plugin/relativeTime';
+import { ListOfSigners } from "../ListOfSigners/ListOfSigners";
+
+dayjs.extend(relativeTime);
+
+type RowType = {
+  id: string;
+  title: string;
+  signatures: string;
+  deadline: string;
+  proposalId: string;
+  item: ProposalType;
+};
+
+const COLUMNS: ColumnOptions<RowType>[] = [{
+  id: 'id',
+  title: 'Proposal ID',
+  }, {
+  id: 'title',
+  title: 'Title',
+  cellRenderer: (row) => <Link href={`https://sovryn.app/bitocracy/${row.proposalId}`} text={truncate(row.title, 20)} openNewTab />,
```

Comment:
Is there any way to detect if you're on testnet? Because the app should work on testnet as well but links always lead to mainnet which is incorrect.